packages/components/src/panel/stories/index.js

```diff
@@ -56,6 +57,27 @@ Default.args = {
 	),
 };
 
+/**
+ * `PanelRow` is a generic container for rows within a `PanelBody`.
+ * It is a flex container with a top margin for spacing.
+ */
+export const _PanelRow = Template.bind( {} );
```

Comment:
I added this story to clarify what the default styling of PanelRow looks like. I don't quite understand why it is `space-between`, but that's something we'll have to keep for back compat. Also the 8px margin-top seems way too narrow.

Though I'm wondering whether we can somehow salvage and update this component to use as #43423 🤔 Something to think about!

Reply:
Good point! I think that `Panel` overall needs a refresh (or a deprecation), especially since more and more of our UI uses `ToolsPanel`. Something that we could at some point discuss with @jasmussen